docs/src/architectures/alchemical-model.rst

```diff
@@ -0,0 +1,46 @@
+.. _architecture-alchemical-model:
+
+Alchemical Model
```

Comment:
Do we want to name this the "alchemical model"? Or is there a better name for this architecture? `alchemical-soap-bpnn` could be an option, or just `alchemical-soap-nn` since the BP part is not very central

Reply:
I would go with alchemical model for now, since this name was mentioned in the lab and in the community multiple times and thus should be recognisable more by this name. IMO we may need the renaming later, if another _alchemical_ architecture would appear and cause ambiguity

Reply:
I agree that it's probably the best name for now